src/views/templates/details/TemplatePageTitle.tsx

```diff
@@ -22,8 +33,15 @@ const TemplatePageTitle: React.FC<TemplatePageTitleTitleProps> = ({ template })
   const lastNamespacePath = useLastNamespacePath();
   const { isCommonTemplate, hasEditPermission } = useEditTemplateAccessReview(template);
 
+  const isSidebarEditorDisplayed =
+    templateTabsWithYAML.find((tab) =>
```

Comment:
u can use the useHistory hook here instead of window

location.pathname.includes(`/templates/${template?.metadata?.name}`)

wont meet the conditions of all 3 ?

Reply:
> u can use the useHistory hook here instead of window

Hmm, I didn't use it when implementing the same for VMs and everyone was fine with that ;) 
Noo, no worries, I'll give it another try. I know using this hook didn't work for me for some cases - it provided an incorrect url, not the actual one. Probably this is why I better didn't use it when working with VMs, not sure.

> location.pathname.includes(/templates/${template?.metadata?.name})

> wont meet the conditions of all 3 ?

Using this condition would lead to displaying YAML switcher ALWAYS, and that's what we don't want to. 

Not sure how I can simplify these conditions. Maybe it would be more efficient to do the opposite: to check if the url does NOT contain a specific url substring - for YAML tab, and if not, then simply to display the switcher. Because here for templates tabs, there are more pages with the switcher than those without it. WDYT?